dotrix_pbr/src/skeletal.rs

```diff
@@ -17,10 +17,22 @@ pub struct Entity {
     pub mesh: Id<Mesh>,
     /// Skin asset ID
     pub skin: Id<Skin>,
-    /// Texture asset ID
+    /// Albedo texture asset ID
     pub texture: Id<Texture>,
     /// Albedo color
     pub albedo: Color,
+    /// Roughness texture asset ID
+    pub roughness_texture: Id<Texture>,
+    /// Roughness (Random scatter)
+    pub roughness: f32,
+    /// Metallic texture asset ID
+    pub metallic_texture: Id<Texture>,
+    /// Metallic (reflectance)
+    pub metallic: f32,
+    /// Ambient occulsion texture asset ID
+    pub ao_texture: Id<Texture>,
+    // Ambient occulsion
```

Comment:
Please, update it to triple slash for documentation: `///`

Reply:
Done